common/client.go

```diff
@@ -81,6 +81,22 @@ func (c *DatabricksClient) AccountClient() (*databricks.AccountClient, error) {
 	return acc, nil
 }
 
+func (c *DatabricksClient) WorkspaceOrAccountRequest(accCallback func(*databricks.AccountClient) error, wsCallback func(*databricks.WorkspaceClient) error) error {
```

Comment:
Micronit: can we name it AccountOrWorkspaceRequest (since the account callback comes first, and the workspace callback comes second)?